cli/wallet/nep17.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/nspcc-dev/neo-go/cli/flags"
 	"github.com/nspcc-dev/neo-go/cli/options"
 	"github.com/nspcc-dev/neo-go/cli/paramcontext"
+	cliSmartcontract "github.com/nspcc-dev/neo-go/cli/smartcontract"
```

Comment:
Package names are all lowercase, so it's better to leave it the same way for aliases.